.github/workflows/osrm-backend-docker.yml

```diff
@@ -0,0 +1,30 @@
+name: build and publish container image
+on:
+  create:
+    tags:
+      - v*
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:       
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+
+      - name: Log in to GitHub Docker Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build container image
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          file: ./docker/Dockerfile
```

Comment:
To support debug/assertion builds like the old hook, you'll need to pass build-args here
https://github.com/Project-OSRM/osrm-backend/blob/34877482e678e44d995454586191ee39d5c31c4b/docker/hooks/build#L9